app/helpers/navigation_helper.rb

```diff
@@ -137,18 +140,20 @@ def in_subcontroller?(page, current_page)
 
   # used for activating tabs in ontology view
   def in_metadata?
-    ontology_nav_metadata.map { |m| m[1][-1].to_s }.include?(controller_name)
+    ontology_nav_metadata.map { |m| m.last.to_s }.
+      include?(controller_name)
   end
 
   protected
 
   def ontology_nav_metadata
+    resource = resource_chain.last
     [
-      ['Projects',         [*resource_chain, :projects]],
-      ['Categories',       [*resource_chain, :categories]],
-      ['Tasks',            [*resource_chain, :tasks]],
-      ['License Models',   [*resource_chain, :license_models]],
-      ['Formality Levels', [*resource_chain, :formality_levels]]
+      ['Projects',         locid_for(resource, :projects), :projects],
+      ['Categories',       locid_for(resource, :categories), :categories],
+      ['Tasks',            locid_for(resource, :tasks), :tasks],
+      ['License Models',   locid_for(resource, :license_models), :license_models],
+      ['Formality Levels', locid_for(resource, :formality_levels), :formality_levels]
```

Comment:
Metrics/LineLength: Line is too long. [85/80]<br>Style/TrailingComma: Put a comma after the last item of a multiline array.